### DISCUSSION TITLE
0.5.6 Servicing Release is now LIVE!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#765] [Announcements]
## Project Reunion 0.5.6 is now available
Project Reunion 0.5.6 is a small, stable servicing update that fixes a few more critical bugs.
If you're building an app with Project Reunion for the first time, follow the [installation instructions here](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fwindows%2Fapps%2Fproject-reunion%2Fget-started-with-project-reunion&data=04%7C01%7CDina.Ayman%40microsoft.com%7C8f34f9acd0494dca90ea08d905d25f9d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637547223445441402%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=BcFAjUkUuPQZkWTF9eCr%2BTI%2FNdHu6Wfvu2DtiF3u8jY%3D&reserved=0).
If you have an app that's already using any version of Project Reunion 0.5, you can update to the latest release by [following the instructions here](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fwindows%2Fapps%2Fproject-reunion%2Fupdate-existing-projects-to-the-latest-release&data=04%7C01%7CDina.Ayman%40microsoft.com%7C8f34f9acd0494dca90ea08d905d25f9d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637547223445451359%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=klznIUa984Sz7HKcSjv0HNjgvawh28h4ZaxiiTv72ZU%3D&reserved=0).

### Bugs fixed in Project Reunion 0.5.6:
•	Metadata is missing from attributes. [The bug fixed can be found here](https://github.com/microsoft/ProjectReunion/issues/767#issue-869085169).
•	Crashes caused by XamlCompiler determining all types are not a part of WinUI framework. [The bug fixed can be found here](https://github.com/microsoft/ProjectReunion/issues/768).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/ProjectReunion/releases/tag/0.5.6'>0.5.6 Servicing Release is now LIVE!</a>.</em>

**Comment by dotMorten** [comment]
Where do I find more information on these issues? 

- Metadata is missing from attributes.
- Crashes caused by XamlCompiler determining all types are not a part of WinUI framework.

**Reply by dayman40** [reply]
Hi @dotMorten, 

The release notes just got updated to include the links for the two bugs that were fixed. Please let us know if you have any questions. 

Thanks,
Dina.

**Reply by riverar** [reply]
@dayman40 The issues you linked were just created (guess these were internal bugs) and are both low quality/missing critical details to inform developers on how critical this update really is.

Then you deleted my comment on issue #767.
![image](https://user-images.githubusercontent.com/475132/116290209-17942200-a748-11eb-9165-05203df8a39a.png)

I'm not mad, but am disappointed in how poorly this release was handled. I hope some lessons were learned for future releases!

**Reply by dayman40** [reply]
Hi @riverar, 
 
I hope you're having a great day! :) I would love clarify with you that it was accidently removed and we apologize for that. The reason why both bugs were just created is because they were tracked internally for customers but in the future we will make sure we publicize our fixed bugs going forward. The were both just edited to be clearer. Please let us know if you have any questions or need help with anything.

Best,

**Reply by dotMorten** [reply]
>  The reason why both bugs were just created is because they were tracked internally for customers

I'm assuming you soon will be tracking all the bugs here from the get-go, instead of only internally so we are aware of any issues that might affects us as well?

**Reply by andrewleader** [reply]
Yep @dotMorten, that's the plan (we realized our mistake and are learning from it, going forward we'll post bugs publicly when they're found from the get-go) 😊

(Note that we still might mess up in the future, changing the way we work can be difficult, but we are working on changing 😊)

**Reply by dotMorten** [reply]
@andrewleader Sounds great! Thanks for listening and learning.
